scenes/globals/pause/pause_overlay.gd

```diff
@@ -0,0 +1,18 @@
+# SPDX-FileCopyrightText: The Threadbare Authors
+# SPDX-License-Identifier: MPL-2.0
+extends CanvasLayer
+
+
+func _unhandled_key_input(event: InputEvent) -> void:
+	if event.is_action_pressed("ui_cancel"):
```

Comment:
Can we have a "pause" action?, otherwise, if we use a joystick it'll pause with `Sony Circle/Xbox B/Nintendo A` by default.